src/main/java/reposense/git/GitRevList.java

```diff
@@ -19,9 +19,13 @@
     private static final String REVISION_PATH_SEPARATOR = " -- ";
 
     /**
-     * Returns the latest commit hash inclusive and until the end of the day of {@code date},
-     * with {@code ZoneId} taken into account.
-     * Returns an empty {@code String} if {@code date} is null, or there is no such commit.
+     * Returns the latest commit hash at {@code branchName} before {@code date}.
+     * Returns an empty {@link String} if {@code date} is null, or there is no such commit.
```

Comment:
If we're standardizing to `{@code String}` then there are couple of instances of leftover `{@link String}`